Support Swift 5, Remove Warring.

dongdongpc opened the pull request.

Commit:
Fix error in swift 4.2

neodave commented:
I would be very interested in this

neodave commented:
The automated 4.2 code update works perfectly except for the UIAlertControllers. dongdongpc has the correct code for that in his fix. This should be simple to do

tejas-ardeshna commented:
@dillidon Please merge this PR

Commit:
1.Remove "Public"
2.Add Switch "@unknown default:fatalError()"
3.Remove deprecated Method, Support Swift 5

Commit:
Edit Readme File

Commit:
1

Commit:
2

Commit:
2

Commit:
remove

Commit:
fix error

Commit:
fix error

Commit:
1

dongdongpc committed:
Delete Alerts&Pickers.podspec

shengyang998 reviewed (changes requested):
Note that you need to remove `Alerts&Pickers.podspec` from project to make it compile successfully.

Commit:
remove pod file

furkanvijapura reviewed (approved):
Done

james-ff commented:
+1
